File: MS.NetProject/Backend/Modal/AddOrder.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace RapidRoute.Modal
{
    public class AddOrder
    {
        public string Name { get; set; }
        public string PhoneNumber { get; set; }
        public string Email { get; set; }
        public string Pickup { get; set; }
        public string Destination { get; set; }

        public DateOnly ODate { get; set; }

        public DateOnly DDate { get; set; }

        public string TrackingID { get; set; }

        [ForeignKey("Customer")]
        public int CustomerId { get; set; }

        [ForeignKey("Agent")]
        public int AgentId { get; set; }
    }
}

File: AdvJavaProject/Backend/NinjaCourier/src/main/java/com/NinjaCourier/AddModal/AddOrder.java
package com.NinjaCourier.AddModal;

public class AddOrder {
	
	public String name;
	public String phoneNumber;
	public String Email ;	
	public String pickup;
	public String destination;
	public int customerId;
	public int agentId;
	
	public String getName() {
		return name;
	}
	public void setName(String name) {
		this.name = name;
	}
	public String getPhoneNumber() {
		return phoneNumber;
	}
	public void setPhoneNumber(String phoneNumber) {
		this.phoneNumber = phoneNumber;
	}
	public String getEmail() {
		return Email;
	}
	public void setEmail(String email) {
		Email = email;
	}
	public String getPickup() {
		return pickup;
	}
	public void setPickup(String pickup) {
		this.pickup = pickup;
	}
	public String getDestination() {
		return destination;
	}
	public void setDestination(String destination) {
		this.destination = destination;
	}
	public int getCustomerId() {
		return customerId;
	}
	public void setCustomerId(int customerId) {
		this.customerId = customerId;
	}
	public int getAgentId() {
		return agentId;
	}
	public void setAgentId(int agentId) {
		this.agentId = agentId;
	}
	
	

}
